pkg/crc/machine/machine.go

```diff
@@ -351,7 +351,7 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 	ocConfig := oc.UseOCWithConfig(startConfig.Name)
 	if !exists {
-		if err := configureCluster(ocConfig, sshRunner, pullSecret); err != nil {
+		if err := configureCluster(ocConfig, sshRunner, crcBundleMetadata.ClusterInfo.BaseDomain, pullSecret, instanceIP); err != nil {
```

Comment:
If the user changes their proxy settings after creating the VM, they will have to go through `crc delete` first. Need to keep that in mind to document it ;)